torchaudio/sox_effects.py

```diff
@@ -20,7 +21,7 @@ def effect_names():
 
 
 @_audio_backend_guard("sox")
-def SoxEffect():
+def SoxEffect() -> Any:
```

Comment:
Instead of putting `Any` here, let's not deal with this case in this PR. We didn't have typing here, so it wouldn't be a regression to omit this one from this PR :)